vector/src/main/java/im/vector/app/core/services/CallRingPlayer.kt

```diff
@@ -94,15 +96,16 @@ class CallRingPlayerIncoming(
 }
 
 class CallRingPlayerOutgoing(
-        context: Context
+        context: Context,
+        private val callManager: WebRtcCallManager
 ) {
 
     private val applicationContext = context.applicationContext
 
     private var player: MediaPlayer? = null
 
     fun start() {
-        applicationContext.getSystemService<AudioManager>()?.mode = AudioManager.MODE_IN_COMMUNICATION
+        callManager.setAudioModeToCallType()
```

Comment:
another option could be to use the `RingtoneManager` but given that we don't already use it I'm assuming we'd prefer not to!